### PR TITLE
Update header in README.md

### LINKED ISSUE #8: Add Emerson to header ofREAME.md
Get it done!

### PR DESCRIPTION
**Pull request by andrew-scoppa**
added Emerson

This PR closes #8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# GitHub for Developers
+# GitHub for Emerson Developers
```
Comment (andrew-scoppa) [inline README.md:1]: Great work!

Comment (bjhans) [reply README.md:1]: 👍

**Review by bjhans — APPROVED** [review]
tada

**Review by bjhans — CHANGES_REQUESTED** [review]
does this block it?

**Review by bjhans — APPROVED** [review]
Approving it again

**Comment by gaherrman** [comment]
I would change Emerson to Fisher

**Comment by msarens** [comment]
Looks Good! 👍

File: README.md (hunk 1 above)
Comment (msarens) [reply README.md:1]: 👍

**Pull request merged**